### ISSUE TITLE
Unknown devices for air quality

### ISSUE THREAD

**Issue by britiger** [Kane610/deconz#552]
I have added [Tuya Smart Air Housekeeper](https://zigbee.blakadder.com/Tuya_DCR-KQG.html) to my deconz, which should already been merged in #408. But in deconz or HA seems to have changed something.

With the last home assistant release 2024.1 I get the following warnings:
```
2024-02-02 20:39:16.165 WARNING (MainThread) [pydeconz.models] Unsupported device type ZHAFormaldehyde
2024-02-02 20:39:16.170 WARNING (MainThread) [pydeconz.models] Unsupported device type ZHACarbonDioxide
```

With older versions of HA with pydeconz <114, I get also the missing particlematter device which alread solved in #531

**Comment by Kane610** [comment]
Can you enable debug logging and share the whole device description for those endpoints?

**Comment by britiger** [comment]
Thanks for the very fast response.

This is the documentation: https://dresden-elektronik.github.io/deconz-rest-doc/devices/tuya/_TZE200_dwcarsat_air_sensor/#smart-air-house-keeper

Here I start HA/pydeconz in debug mode:
[home-assistant_deconz_2024-02-02T21-15-49.818Z.log](https://github.com/Kane610/deconz/files/14145721/home-assistant_deconz_2024-02-02T21-15-49.818Z.log)

In phoscon I only can see temp and hum values. 

If you need more information please let me know.

**Comment by Kane610** [comment]
Sorry for the delay but I've put up a PR to support both sensors in Home Assistant now

**Comment by Kane610** [comment]
This has now been merged and will be available in home assistant 2024.6.0 in a couple of weeks
